README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I think I'd have to use a helper method. They make some or all of the controller's methods available for use in the view so that would be understanding. Also, this question's examples are hilarious.
 ```
```

Comment:
:+1: Correct, although to be explicit, helper methods are useful for rendering information in a view based on certain conditions.